pymysqlreplication/binlogstream.py

```diff
@@ -134,7 +135,8 @@ def __init__(self, connection_settings, server_id, ctl_connection_settings=None,
                  freeze_schema=False, skip_to_timestamp=None,
                  report_slave=None, slave_uuid=None,
                  pymysql_wrapper=None,
-                 fail_on_table_metadata_unavailable=False):
+                 fail_on_table_metadata_unavailable=False,
+                 representation='HUMAN_READABLE'):
```

Comment:
I think here I will directly put as parameter the representation class to allow anyone to implement another without touching the lib code.